plugins/modules/redfish_storage_volume.py

```diff
@@ -679,6 +878,11 @@ def main():
         "volume_id": {"required": False, "type": "str"},
         "oem": {"required": False, "type": "dict"},
         "initialize_type": {"type": "str", "required": False, "choices": ['Fast', 'Slow'], "default": "Fast"},
+        "apply_time": {"required": False, "type": "str", "choices": ['Immediate', 'OnReset']},
+        "reboot_server": {"required": False, "type": "bool", "default": 'False'},
```

Comment:
No need to mention required False, by default it's False. mention only for True. please check and update, will be more readable.

Reply:
Argument 'job_wait' in argument_spec defines default as (None) but documentation defines default as (False)

This error will be thrown in sanity.